lisa/util/process.py

```diff
@@ -235,3 +241,67 @@ def _filter_sudo_result(self, raw_input: str) -> str:
             raw_input = "".join(lines[1:])
             self._log.debug(f'found error message in sudo: "{lines[0]}"')
         return raw_input
+
+
+def local_execute(
```

Comment:
It can be done by use a local node. Refer to https://github.com/microsoft/lisa/blob/main/lisa/runners/legacy_runner.py#L81
If you fell the two lines are ugly, refer to below to create a helper method to return a local node. So it doesn't need to construct the mockup runbook everytime.
https://github.com/microsoft/lisa/blob/main/lisa/runners/legacy_runner.py#L81

Reply:
You posted the same link twice?

Reply:
It doesn't need to copy the code, and it leverages all benefits of lisa, for example, using node.tools to use any tool.